Formula/gomplate.rb

```diff
@@ -15,7 +15,9 @@ class Gomplate < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "3703f68a2d7c4598da6a0ec112ed38bb79a61073ffc161728721dd2e61895e9b"
   end
 
-  depends_on "go" => :build
+  # Required latest https://pkg.go.dev/go4.org/unsafe/assume-no-moving-gc
+  # Try to switch to the latest go on the next release
+  depends_on "go@1.18" => :build
```

Comment:
FYI once https://github.com/hairyhenderson/gomplate/pull/1469 merges I'll release gomplate 3.11.2, which should make this change unnecessary